client/src/services/ApiService.ts

```diff
@@ -72,6 +72,9 @@ export default {
   fetchMetar(flightId: string) {
     return apiClient.get("flights/admin/fetch-metar/" + flightId);
   },
+  changeFlightProps(flightId: string, flightProps: Object) {
```

Comment:
type: object is bad practice. better type the object so that you know what it is.

Reply:
I thought about that but that means I need to implement a copy of the type Flight but w only optional props...
Is there any nice way to do that instead of only copying every thing?